CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 - `messageLabelInsets(for:indexPath:messagesCollectionView` method to `MessagesLayoutDelegate`. 
 [#162](https://github.com/MessageKit/MessageKit/pull/162) by [@SD10](https://github.com/SD10).
 
+-`animationBlockForLocation(message:indexPath:messagesCollectionView)` method to `LocationMessageDisplayDelegate` to customize the display animation of the location message's map
```

Comment:
1. Can you add a space so we have a proper bullet point 😉 
2. Can you end every sentence with a period?
3. Can you mark the PR number with the `#` character?

Very nitty, I can change them after merge if you'd like. I will add these things to the CHANGELOG guidelines as well.

Reply:
I will change them, don't worry. Is good to not forget later on 😁